packages/cdktf-cli/lib/checkpoint.ts

```diff
@@ -94,6 +102,20 @@ export async function ReportRequest(reportParams: ReportParams): Promise<void> {
     reportParams.os = os.platform();
   }
 
+  const ci: string | false = ciDetect();
+  if (!reportParams.userId && !ci) {
+    reportParams.userId = machineIdSync();
+  }
+
+  if (ci) {
+    reportParams.ci = ci;
+  }
+
+  const gitRemote = await gitRemoteOriginUrl();
+  const hash = crypto.createHash("md5");
+  hash.update(gitRemote);
+  reportParams.gitRepoId = hash.digest("hex").slice(0, 32).toUpperCase();
```

Comment:
Why is it truncated?

Reply:
I was unsure on how long it will be and I thought 32 should be enough to not run into collisions.

Reply:
Asking stackoverflow yields

> A 128-bit MD5 hash is represented as a sequence of 32 hexadecimal digits

https://stackoverflow.com/questions/3394503/maximum-length-for-md5-input-output

Reply:
huh, yeah seems like I don't need the slice then